pinot-broker/src/main/java/org/apache/pinot/broker/api/resources/PinotBrokerRouting.java

```diff
@@ -61,8 +69,23 @@ public class PinotBrokerRouting {
       @ApiResponse(code = 200, message = "Success"),
       @ApiResponse(code = 500, message = "Internal server error")
   })
-  public String buildRouting(
+  public String buildRouting(@Context HttpHeaders headers,
```

Comment:
(nit, convention) We usually put this as the last argument

Reply:
Noted. Fixed it.